.gitignore

```diff
@@ -10,4 +10,6 @@ reactHooksSample.jsx
 coverage
 spec/__snapshot__
 database/data/datasets
-newrelic.js
\ No newline at end of file
+newrelic.js
```

Comment:
Keep in mind if you store your new relic key in an environment variable you won't have to git ignore

Reply:
very good point!